docs/Playing/how-to-play.rst

```diff
@@ -1,224 +1,366 @@
+tile. This citizen can then be transferred to a specialist. There are three specialists: Taxmen (collect
+gold), Entertainers (create happiness), or Scientist (create bulbs). By default, Freeciv21 will add an
+Entertainer when you take a citizen off the land.
+
+If you click on the central tile of the land (the "City Center"), the citizens will be rearranged to maximize
+food production. You will also want to look at the level of food in the Granary and the amount of surplus food
+the city is producing each turn. The city will lose excess food at turn change. For example: The Granary in
+the city has 18/20 food (needs two food to grow to next city size) and the city is currently producing +4 food
+surplus each turn. This means that at turn change you will lose 2 food as waste at turn change. This is a good
+opportunity to move the citizens around to get food to +2 surplus. This could be accomplished by taking a
+citizen off a tile producing 2 food and turn it into a Taxman for a turn to get gold. At turn change, open the
+city and restore the citizen to farming. This kind of individual city management style is called
+"micro-management" and is a very powerful mechanism of game play.
+
+The golden rule of taking care of a city is that there should be at least as many happy citizens as unhappy
+citizens. A city where this is not so falls into disorder. Such cities are labeled with a raised fist or a
+lightning bolt (depending on the tileset). Take care not to let this happen to any of your cities, as cities
+in disorder produce nothing, and are prone to revolt.
+
+Interacting with Other Players (Diplomacy)
+------------------------------------------
+
+When one of your units first meets a unit of another nation, or finds one of their cities (or equally if
+they find you), a basic contact is established between the two nations. This provides each with basic
+intelligence about the other, which can be accessed from the
+:numref:`Nations and Diplomacy View <Nations and Diplomacy View Button>`.
+
+This communication will lapse after a ruleset defined number of turns with no contact. Establishing an embassy
+will give a more permanent communication channel, as well as more advanced intelligence such as details of
+technology. Embassies are one-way, the nation hosting the embassy receives no benefit and once established,
+cannot be revoked.
+
+If you are in contact with another player, then you can arrange a diplomatic meeting. From the
+:numref:`Nations and Diplomacy View <Nations and Diplomacy View Button>` , this is done by selecting the
+nation with whom you wish to meet and clicking :guilabel:`Meet`. If the entry under the embassy column is not
+blank and the other player is connected (or is a server AI) then a treaty dialog will pop up.
+
+In this dialog you can negotiate an exchange of assets (maps, vision, advances, cities, or gold), Embassies,
```

Comment:
Inconsistent capitalization of "embassy", it isn't capitalized at the beginning of this section, but is capitalized here